terra-app-local.sh

```diff
@@ -117,11 +117,9 @@ install() {
     echo ""
     echo "Once the pod is RUNNING, do the following to open the app:"
     echo ""
-    echo "1. Add an entry to your /etc/hosts:"
-    echo "     ${_minikube_ip}  ${_hostname}"
+    echo "1. Run minikube service list to get service namespace and service name"
```

Comment:
most likely we are hitting this issue: https://github.com/kubernetes/minikube/issues/7344
I still change to this way (instead of other workaround mentioned in that thread) because I don't need to kept updating /etc/hosts...